docs/designers-developers/developers/README.md

```diff
@@ -6,40 +6,40 @@ The new editor is highly flexible, like most of WordPress. You can build custom
 
 The editor is about blocks, and the main extensibility API is the Block API. It allows you to create your own static blocks, dynamic blocks rendered on the server and also blocks capable of saving data to Post Meta for more structured content.
 
-If you want to learn more about block creation, the [Blocks Tutorial](../../../docs/designers-developers/developers/tutorials/block-tutorial/readme.md) is the best place to start.
+If you want to learn more about block creation, the [Blocks Tutorial](/docs/designers-developers/developers/tutorials/block-tutorial/readme.md) is the best place to start.
 
 ## Extending Blocks
 
 It is also possible to modify the behavior of existing blocks or even remove them completely using filters.
 
-Learn more in the [Block Filters](../../../docs/designers-developers/developers/reference/hooks/block-filters.md) section.
+Learn more in the [Block Filters](/docs/designers-developers/developers/reference/hooks/block-filters.md) section.
 
 ## Extending the Editor UI
 
 Extending the editor UI can be accomplished with the `registerPlugin` API, allowing you to define all your plugin's UI elements in one place.
 
 Refer to the [Plugins](https://github.com/WordPress/gutenberg/blob/master/packages/plugins/README.md) and [Edit Post](https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/README.md) section for more information.
 
-You can also filter certain aspects of the editor; this is documented on the [Editor Filters](../../../docs/designers-developers/developers/reference/hooks/editor-filters.md) page.
+You can also filter certain aspects of the editor; this is documented on the [Editor Filters](/docs/designers-developers/developers/reference/hooks/editor-filters.md) page.
 
 ## Meta Boxes
 
 **Porting PHP meta boxes to blocks or sidebar plugins is highly encouraged!**
 
-Discover how [Meta Box](../../../docs/designers-developers/developers/backward-compatibility/meta-box.md) support works in the new editor.
+Discover how to work with [Meta Boxes](/docs/designers-developers/developers/tutorials/metabox/readme.md) in the new editor.
```

Comment:
I think this should still link to the meta-box backcompat docs because it explains how legacy meta boxes currently work and how we handle backward compatibility.

Alternately, we should at least make sure to link to _both_ files.

Reply:
Updated to include two paragraphs, tying in the encouragement to the new tutorials and rewording the previous one for clarity, ie. "existing meta boxes"